packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,83 @@
+default.target=.
+
+## all templates to apply on target deployment env
+nuxeo.template.includes=default,docker,s3binaries,postgresql,explorer-sitemode,explorer-virtualadmin
```

Comment:
assumed the docker template is what we wanted for logs management

Reply:
there is also a docker-json variant if needed

Reply:
thanks, i saw it (see initial description), i'm just not sure what will match best the target env, so i went for the docker one for now